sci-libs/vtk/vtk-9.2.5.ebuild

```diff
@@ -447,7 +447,7 @@ src_configure() {
 			-DVTK_DEBUG_MODULE=ON
 			-DVTK_DEBUG_MODULE_ALL=ON
 			-DVTK_ENABLE_SANITIZER=ON
-			-DVTK_EXTRA_COMPILER_WARNINGS=ON
+#			-DVTK_EXTRA_COMPILER_WARNINGS=ON # bug #891829
```

Comment:
Does this option enable extra warnings, or does it make the compiler treat more warnings as errors? Extra warnings is fine, but usually we disable anything that can cause warnings to be treated as errors (i.e.  `Werror` stuff) because having warnings treated as errors is not very useful for the end-user.

Reply:
See also: https://devmanual.gentoo.org/ebuild-writing/common-mistakes/index.html#-werror-compiler-flag-not-removed

Reply:
It adds `-Werror=undef`, but not a general `-Werror`. The complete set of added gcc flags can be seen in https://github.com/Kitware/VTK/blob/master/CMake/vtkCompilerExtraFlags.cmake#L6

Reply:
Maybe we could just filter this flag out?

Reply:
> Maybe we could just filter this flag out?

Yeah, I think that is the easiest solution.

Reply:
I change the PR then. Didn't thought about this in the first place, only when I wrote the comment 15 minutes ago O.o

Reply:
Looks like I can't filter these with the `filter-flags` function from `flag-o-matic.eclass`. Need to create a patch for it.